app/templates/marketing/terms_of_service.html

```diff
@@ -41,21 +41,45 @@ <h4 class="ui header" id="issue_tracker">Issues or Concerns?</h4>
 
         <h1 class="ui dividing header" id="code_of_conduct">Code of Conduct</h1>
 
+        <p>As a Python community we are beholden to the Python Software Foundation (PSF)
+           <a href="https://www.python.org/psf/codeofconduct/">Code of Conduct</a>.
+        </p>
+        <p>This is extended by our own PySlackers Code of Conduct detailed below:</p>
+
         <h2 class="ui header" id="our_pledge">Our Pledge</h2>
 
         <p>In the interest of fostering an open and welcoming environment, we as
-            contributors and maintainers pledge to making participation in our
+            contributors and maintainers pledge to make participation in our
             project and
             our community a harassment-free experience for everyone, regardless
             of age, body
-            size, disability, ethnicity, gender identity and expression, level
+            types, disability, ethnicity, gender identity and expression, level
             of experience,
             nationality, personal appearance, race, religion, or sexual identity
             and
             orientation.</p>
 
         <h2 class="ui header" id="our_standards">Our Standards</h2>
 
+        <p>As a community we are:</p>
+        <ul>
+            <li>
+                <a href="https://www.python.org/psf/codeofconduct/#open">Open</a> to collaboration, change and
+                constructive comment and criticism from any and all members of the community
```

Comment:
Lots of `and` makes it a bit hard to understand. But no idea of a better formulation